weave/scorers/llm_as_a_judge_scorer.py

```diff
@@ -0,0 +1,85 @@
+import json
+from typing import Any, Type, Union
+
+import jsonschema
+from pydantic import BaseModel, Field
+
+from weave.scorers.scorer_types import LLMScorer
+from weave.trace.op import op
+
+
+class LLMAsAJudgeScorer(LLMScorer):
```

Comment:
Inheriting from `LLMScorer` seems like overkill. The LLMScorer was written by growth team and just binds litellm methods to the class (and a model_id). Curious why this was chosen